DIGITfiles/ParseSangerResults.py

```diff
@@ -14,13 +15,44 @@
 queriesWorkingSet = {}
 
 
+def sameQuery(queryS, queryOG):
```

Comment:
Don't forget your docstrings explaining the function purpose, parameters, return values, exceptions, etc